src/modules/trayicon/libkvitrayicon.cpp

```diff
@@ -310,15 +302,15 @@ void KviTrayIconWidget::fillContextPopup()
 	if(g_pApp->topmostConnectedConsole())
 	{
 		m_pAwayMenuId->setVisible(true);
-		m_pAwayPopup->clear();
+		m_awayPopup.clear();
 
-		QAction * pAllAway = m_pAwayPopup->addAction(*(g_pIconManager->getSmallIcon(KviIconManager::Away)), __tr2qs("Away on All"), this, SLOT(doAway(bool)));
+		QAction * pAllAway = m_awayPopup.addAction(*(g_pIconManager->getSmallIcon(KviIconManager::Away)), __tr2qs("Away on All"), this, SLOT(doAway(bool)));
```

Comment:
:+1: 
this icon tray away controller, is utterly broken currently, fingers crossed this fixes it.